### PR TITLE
Improved comments and indentations

### PR DESCRIPTION
**Pull request by ankitsaxena21**
I improved the comments and indentations in python files.
I added lines, removed extra blank spaces and modified comments as per the [The official PEP8 python style guide](https://www.python.org/dev/peps/pep-0008/).
I used **is** keyword in place of **==**.
This will make the codes matching with the standard guidelines.

### CODE REVIEW

File: callgraph/core.py [hunk 1]
```diff
@@ -24,15 +26,20 @@ def AddCommand(self, command):
...
+
     def __eq__(self, other):
         return other is not None and self.number == other.number and self.text == other.text and self.terminating == other.terminating
 
 # Connection between two nodes.
-# dst is the name of the target node, kind is the type of connection, line_number is the line where the call/goto happens.
+# dst is the name of the target node, kind is the type of connection,
+# line_number is the line where the call/goto happens.
+
+
```
Comment (lupino3) [inline callgraph/core.py]: Could you remove those spaces between the comment about the namedtuple and the definition of the namedtuple itself?

File: callgraph/core.py [hunk 2]
```diff
@@ -119,19 +127,23 @@ def _MarkExitNodes(self):
...
                     q.append(self.nodes[connection.dst])
-            
 
-    # Adds to each node information depending on the contents of the code, such as connections
-    # deriving from goto/call commands and whether the node is terminating or not.
+    # Adds to each node information depending on the
+    # contents of the code, such as connections
+    # deriving from goto/call commands and
+    # whether the node is terminating or not.
+
```
Comment (lupino3) [inline callgraph/core.py]: Please remove this line, I think it's nice when comments are attached to the thing they are explaining.

File: callgraph/core.py [hunk 3]
```diff
@@ -246,10 +262,10 @@ def Build(input_file, log_file=sys.stderr):
 
         return call_graph
 
-    # Creates a call graph from an input file, parsing the file in blocks and creating
-    # one node for each block. Note that the nodes don't contain any information that
-    # depend on the contents of the node, as this is just the starting point for the
-    # processing.
+    # Creates a call graph from an input file, parsing the file in blocks and
+    # creating one node for each block. Note that the nodes don't contain any \
```
Comment (lupino3) [inline callgraph/core.py]: Please remove this slash.

File: callgraph/core.py (hunk 1 above)
Comment (ankitsaxena21) [reply callgraph/core.py]: Yes, I have removed the spaces.

File: callgraph/core.py (hunk 2 above)
Comment (ankitsaxena21) [reply callgraph/core.py]: I have removed the line 135.

File: callgraph/core.py (hunk 3 above)
Comment (ankitsaxena21) [reply callgraph/core.py]: done.

**Comment by ankitsaxena21** [comment]
Thanks for the correction!
I had made the required changes. Please inform, if any other changes are needed to be made.

**Comment by lupino3** [comment]
Thank you very much. LGTM. Merging it now.

**Pull request merged**